crypto/keyring/keyring.go

```diff
@@ -258,6 +265,11 @@ func (ks keystore) ExportPubKeyArmor(uid string) (string, error) {
 	return crypto.ArmorPubKeyBytes(bz, key.Type()), nil
 }
 
```

Comment:
can we add 

```
var (
 _ Keyring = keystore{}
_ KeyringWithDB = keystore{}
)
```

as type assertions?

Reply:
typically we add this where the type is declared

above `type keystore struct { ...`